src/App.js

```diff
@@ -10,16 +10,20 @@ class App extends Component {
     this.state = {
       parks: [],
       usStates: {},
+      currentUsStateName: 'default',
+      currentUsStateCoord: [],
       currentParksToShow: [],
       vistedParks: [],
       bucketListParks: [],
-      pageStatus: 'landing'
+      pageStatus: 'landing',
+      randomImageClass: `landing-background${Math.floor(Math.random() * 6)}`
```

Comment:
Great job randomizing the landing page image

Reply:
Just reloaded the page a few times to see the different images, they're awesome pictures!